extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1334,6 +1349,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       attributes['reportCreativeGeometry'] = true;
       attributes['isDifferentSourceWindow'] = false;
       attributes['sentinel'] = this.sentinel;
+    } else {
+      Object.assign(attributes, this.nameframeExperimentConfig_);
```

Comment:
Why don't we want this to apply to fluid as well?

Reply:
Ah so I would have expected us to run this in both AdSense and Doubleclick.  Can you add header detection/implementation to AdSense impl as well.

Reply:
Done.